### DISCUSSION TITLE
IPAC2 and antimicro

### DISCUSSION THREAD

**Discussion by tr3ndkill** [AntiMicroX/antimicrox#828] [Q&A]
hi ,. im using antimicro on my arcade cabinet
i have ipac2 as 2 xinput controllers
anti micro recognize it as xinput1 and xinput 2 and the key presses are seen in each window
i thanks changes the buttons to keybord on both xinput devices but as soon as i exit anti micro (cloze it ) it doesnt save the changes and always use the same keybord layout of xinput 1 to xinput 2
how can i define diffrent keys for the two xinput 1,2 devices ?
thanks

**Comment by mrbeauregarde** [comment]
literally the previous Q&A question
